doc/nrf/ug_matter_overview_network_topologies.rst

```diff
@@ -51,6 +46,12 @@ Bridge
 Controller
   |matter_controller_def|
 
+Edge Router
```

Comment:
Isn't Edge Router the same as Border Router? I mean it's ok for me, but I don't know why we introduce another terminology, while Border Router is easier to associate with e.g. Thread Border Router. That's just consideration, I'm leaving it to your decision.

From Matter spec: 

> Border Router A router, also known as Edge Router, that provides routing services between
> two IP subnets (typically, between a hub network and a peripheral network).

Reply:
That's the line @LuDuda used to justify using Edge Router. I think Border Router is immediately associated with OTBR. Edge router is less common and it AP could fit this description. An alternative is to mention Access Point separately.

Reply:
Let it be then. Just let's make sure difference between edge and border between networks will not confuse anyone, as there is none. I thought it's good, because many people are not aware of the Thread network topology and Thread vs Wi-Fi connection, so basically border between those two networks is important for us to highlight. Of course Wi-Fi AP is also border router bridging local network with something bigger, but from this perspective most routers on the world are border routers to the worldwide Internet :)

Reply:
Or we can also just copy the text from spec so `Border Router (also known as Edge Router)`.